packages/schematics/angular/utility/latest-versions.ts

```diff
@@ -8,14 +8,14 @@
 
 /** Retrieve the minor version for the provided version string. */
 function getEarliestMinorVersion(version: string) {
-  const versionMatching = version.match(/^(\d+)\.(\d+)\.*/);
+  const versionMatching = version.match(/^(\d+)\.(\d+)\.(?:\d+)(-\w+)?/);
```

Comment:
Is it possible that `@schematics/angular` could move into `rc` while `@angular/core` is still in `next` resulting in a mismatch for this discovered `prerelease` segment?

Reply:
I don't think so, the FW is always released first.